src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/database/jpa/entity/GoalEntity.kt

```diff
@@ -70,13 +72,21 @@ class GoalEntity(
   @CreatedBy
   var createdBy: String? = null,
 
+  @Column
+  @AuditDisplayName
+  var createdByDisplayName: String? = null,
```

Comment:
Nice work with the annotation. However, I can't see what's stopping this being updated when an update occurs?

Reply:
Oh, yes, good shout!
Hmm ... I'll give that some thought ...